### DISCUSSION TITLE
yarn installation through vagrant error

### DISCUSSION THREAD

**Discussion by alexandreumb** [mastodon/mastodon#29524] [Q&A]
Hello i am doing set up of mastodon  and when running
vagrant up i am getting
![image](https://github.com/mastodon/mastodon/assets/106603120/093f1ed8-db54-4bbe-b51b-0095fd1cca95)
i have tried to do the yarn installation independent from the other as well by running:
vagrant ssh -c "cd /vagrant && yarn install"
but the error is the same, what might the problem be

**Comment by mjankowski** [comment]
Did you resolve this?

If not, can you add some detail about env/versions/etc?
